### PR TITLE
create-ceph-cluster

### PR DESCRIPTION
**Pull request by gnehapk**

### CODE REVIEW

**Comment by gnehapk** [comment]
@cloudbehl Please review.

File: src/modules/cluster/create-cluster/create-ceph-cluster/step-third.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                        </select>
+                    </div>
+                    <button ng-init="createCephClusterCntrl.ascOrder = ''" ng-disabled="createCephClusterCntrl.avaialablEhosTforRole.length === 0" class="btn btn-link" type="button">
+                        <span ng-if="createCephClusterCntrl.ascOrder !== ''" class="fa fa-sort-alpha-desc" ng-click="createCephClusterCntrl.ascOrder = ''">
+                        </span>
+                        <span ng-if="createCephClusterCntrl.ascOrder !== '-'" class="fa fa-sort-alpha-asc" ng-click="createCephClusterCntrl.ascOrder = '-'">
+                        </span>
+                    </button>
+                </div>
+                <!-- <div class="toolbar-pf-action-right">
```
Comment (cloudbehl) [inline src/modules/cluster/create-cluster/create-ceph-cluster/step-third.html]: Can we remove all these? Commented lines!

File: src/modules/cluster/create-cluster/create-ceph-cluster/step-third.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                        <div class="row">
+                            <div class="col-md-offset-6 col-md-6">
+                                <button class="btn btn-primary pull-right" ng-click="createCephClusterCntrl.refreshHostList()">Refresh</button>
+                            </div>
+                        </div>
+                    </div>
+                </div> -->
+            </form>
+
+            <!-- result row -->
```
Comment (cloudbehl) [inline src/modules/cluster/create-cluster/create-ceph-cluster/step-third.html]: Can we remove all these? Commented lines!

File: src/modules/cluster/create-cluster/create-ceph-cluster/step-third.html [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+        </center>
+        <div class="list-group list-view-pf list-view-pf-equalized-column" ng-repeat="host in createCephClusterCntrl.filteredHostForRole=(createCephClusterCntrl.availableHostForRole | filter: createCephClusterCntrl.searchBy | orderBy : createCephClusterCntrl.ascOrder + createCephClusterCntrl.orderBy)">
+            <div class="list-group-item">
+<!--                 <div class="col-md-1">
+                    <input type="checkbox" ng-model="host.checkboxSelected">
+                </div> -->
+                <div class="col-md-3">
+                    <div class="list-view-pf-additional-info-item list-view-pf-additional-info-item-stacked">
+                        <div class="bold-text">{{host.fqdn || "NA"}}</div>
+                        <!-- <h5 class="pull-left counts" ng-modal="host.name">{{host.ip || "NA"}}</h5> -->
```
Comment (cloudbehl) [inline src/modules/cluster/create-cluster/create-ceph-cluster/step-third.html]: Can we remove all these? Commented lines!

File: src/modules/cluster/create-cluster/create-ceph-cluster/step-third.html [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+            <div class="blank-slate-pf">
+                <div class="blank-slate-pf-icon">
+                    <i class="pficon pficon-container-node"></i>
+                </div>
+                <h1>No Hosts Detected.</h1>
+            </div>
+        </center>
+        <div class="list-group list-view-pf list-view-pf-equalized-column" ng-repeat="host in createCephClusterCntrl.filteredHostForRole=(createCephClusterCntrl.availableHostForRole | filter: createCephClusterCntrl.searchBy | orderBy : createCephClusterCntrl.ascOrder + createCephClusterCntrl.orderBy)">
+            <div class="list-group-item">
+<!--                 <div class="col-md-1">
```
Comment (cloudbehl) [inline src/modules/cluster/create-cluster/create-ceph-cluster/step-third.html]: Can we remove all these? Commented lines!

File: src/modules/cluster/create-cluster/create-ceph-cluster/step-second.html [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+                <div class="blank-slate-pf-icon">
+                    <i class="pficon pficon-container-node"></i>
+                </div>
+                <h1>No Hosts Detected.</h1>
+            </div>
+        </center>
+        <div ng-if="createCephClusterCntrl.isDataLoading" class=""></div>
+        <div class="list-group list-view-pf list-view-pf-equalized-column filter-section">
+            <div class="list-group-item">
+                <div class="col-md-1"></div>
```
Comment (cloudbehl) [inline src/modules/cluster/create-cluster/create-ceph-cluster/step-second.html]: can we make then span of col-md-4? why two different divs for making the space?

File: src/modules/cluster/create-cluster/create-cluster-modal/create-cluster-modal.scss [hunk 6]
```diff
@@ -0,0 +1,17 @@
+.tendrl-create-cluster-modal-container{
+
+    ul > li:hover {
+        background-color: #bee1f4;
+        border-color: #bee1f4;
+    }
+
+    .selectedCluster{
+    	background-color: $color-curious-blue!important;
```
Comment (cloudbehl) [inline src/modules/cluster/create-cluster/create-cluster-modal/create-cluster-modal.scss]: Recommended not to use important. If not required can we remove it?

**Comment by cloudbehl** [comment]
@gnehapk can you have a look at the comments?

**Pull request merged**